### PR TITLE
[TACHYON-1428] Documented parts of the tachyon-servers module in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: servers/src/test/java/tachyon/worker/block/allocator/RoundRobinAllocatorTest.java [hunk 1]
```diff
@@ -24,6 +24,12 @@
  * Unit tests for {@link RoundRobinAllocator}.
  */
 public class RoundRobinAllocatorTest extends BaseAllocatorTest {
+
+  /**
+   * Tests that block are allocated in a round robin fashion.
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/allocator/RoundRobinAllocatorTest.java]: typo: block -> blocks

File: servers/src/test/java/tachyon/worker/block/allocator/MaxFreeAllocatorTest.java [hunk 2]
```diff
@@ -24,6 +24,12 @@
  * Unit tests for {@link MaxFreeAllocator}.
  */
 public class MaxFreeAllocatorTest extends BaseAllocatorTest {
+
+  /**
+   * Tests that block are allocated in the storage directory with the most available free space.
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/allocator/MaxFreeAllocatorTest.java]: typo: block -> blocks

File: servers/src/test/java/tachyon/worker/block/allocator/GreedyAllocatorTest.java [hunk 3]
```diff
@@ -24,6 +24,12 @@
  * Unit tests for {@link GreedyAllocator}.
  */
 public class GreedyAllocatorTest extends BaseAllocatorTest {
+
+  /**
+   * Tests that block are allocated in the first storage directory which has enough free space.
```
Comment (yupeng9) [inline servers/src/test/java/tachyon/worker/block/allocator/GreedyAllocatorTest.java]: typo: block -> blocks

**Comment by yupeng9** [comment]
Thanks for the doc improvement. I left some minor comments.

**Comment by HorizonNet** [comment]
@yupeng9 Thanks for the comments. I updated the PR accordingly.

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**